script.js

```diff
@@ -0,0 +1,28 @@
+const daysCount = document.querySelector('.days-count');
+const hoursCount = document.querySelector('.hours-count');
+const minutesCount = document.querySelector('.minutes-count');
+const secondsCount = document.querySelector('.seconds-count');
+const NewYear = '1 Jan 2023';
```

Comment:
Please try to set NewYear data from the browser using Javascript's Date() function. Because It will let your app work for the coming years as well. With the static data you gave the app will work only for 2023, but you can change this so that it works for all the years to come.